client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/JsonEvents.java

```diff
@@ -0,0 +1,28 @@
+package org.infinispan.client.hotrod.event;
+
+import static org.infinispan.client.hotrod.test.HotRodClientTestingUtil.withClientListener;
+import static org.infinispan.commons.dataconversion.MediaType.APPLICATION_JSON;
+
+import org.infinispan.client.hotrod.DataFormat;
+import org.infinispan.client.hotrod.test.SingleHotRodServerTest;
+import org.infinispan.commons.marshall.UTF8StringMarshaller;
+import org.testng.annotations.Test;
+
+/**
+ * @since 9.4
+ */
+@Test(groups = {"functional",}, testName = "client.hotrod.event.JsonEvents")
+public class JsonEvents extends SingleHotRodServerTest {
```

Comment:
Please name it `*Test` or surefire won't run it

Reply:
Done